packages/dev/src/main.ts

```diff
@@ -183,6 +214,7 @@ export class NetlifyDev {
 
     this.#apiToken = options.apiToken
     this.#cleanupJobs = []
+    this.#geolocationConfig = options.geolocation
```

Comment:
let's follow the existing pattern, where the `enabled` boolean lives on `this.#features` and any additional config gets its own top-level key